custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -341,20 +341,25 @@ def run_fit(self):
             print("Success 🎉")
 
     def run_test_predict(self):
-        self.run_mode = RunMode.SCORE
         self.options.code_dir = self.options.output
         self.options.output = os.devnull
+        __target_temp = None
         if self.options.target:
             __tempfile = NamedTemporaryFile()
             df = pd.read_csv(self.options.input)
+            if self.target_type == TargetType.TRANSFORM:
+                target_df = df[self.options.target]
+                __target_temp = NamedTemporaryFile()
+                target_df.to_csv(__target_temp.name, index=False)
```

Comment:
i thought we were going to use a virtual file here? or am I confused.

Reply:
I left this as is since we return a temp file anyway if it fails the check....I guess we can use a virtual file and then add another step to dump to a temp file on failure?

Reply:
and of course that broke something

Reply:
and fixed